### PR TITLE
chore(ci): adding appveyor configuration.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+environment:
+  matrix:
```
Comment (jkuri) [inline .appveyor.yml]: please remove this line

File: .appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    - nodejs_version: "5.0"
```
Comment (jkuri) [inline .appveyor.yml]: and fix this to be without `- ` prefix

File: .appveyor.yml (hunk 1 above)
Comment (hansl) [inline .appveyor.yml]: done

File: .appveyor.yml (hunk 2 above)
Comment (hansl) [inline .appveyor.yml]: done

**Comment by jkuri** [comment]
lgtm, lets see if it will be included automatically in "getting green".

**Comment by hansl** [comment]
It won't :)  That's why I'm spending some time on it.

**Pull request merged**